src/Symfony/Bundle/SecurityBundle/Resources/config/console.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bundle\SecurityBundle\Command\UserPasswordEncoderCommand;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('security.command.user_password_encoder', UserPasswordEncoderCommand::class)
+        ->args([service('security.encoder_factory'), []])
```

Comment:
indention is wrong

Reply:
please use `abstract_arg('encoders user classes')` for the second argument and on a new line

Reply:
Done